src/components/richLink.tsx

```diff
@@ -139,6 +139,10 @@ const richLinkStyles = (format: Format): SerializedStyles => {
 		@media (max-width: 23.4rem) {
 			width: 100%;
 			box-sizing: border-box;
+
+			.js-image {
+				display: none;
+			}
```

Comment:
I'd suggest applying a `css` attribute to the div rather than styling on the `.js-image` class. The purpose of `.js-*` classes is to say "hey, this is a hook for some JS code, it's safe to manipulate (add/remove etc.) because there are no styles hanging off it".